docs/src/usage.md

```diff
@@ -259,7 +259,7 @@ The "{index}`Comment <pair: Comment;Metric>`" field can be used to capture the r
 
 #### Reordering metrics
 
-To reorder metrics, expand the metric in the metric table and use the buttons on the lower left-hand side to move the metric one row higher or lower, or to the top or bottom of the table. The order is saved on the server. Users can temporarily override the default ordering of the metrics by clicking the column headers thus sorting by e.g. name or status.
+To reorder metrics, expand the metric in the metric table and use the buttons on the lower left-hand side to move the metric one row higher or lower, or to the top or bottom of the table. The order is saved on the server. Users can temporarily override the default ordering of the metrics by clicking the column headers thus sorting by, for example, name or status.
```

Comment:
I think there should be a comma before "thus". But it might be more clear to end the sentence with a full stop there and addressing the logic of the sorting fallback in a next sentence.